nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/file/classloader/ClassLoaderUtils.java

```diff
@@ -137,25 +137,17 @@ private static boolean isNotBlank(final String value) {
         return additionalClasspath.toArray(new URL[additionalClasspath.size()]);
     }
 
-    public static String generateAdditionalUrlsFingerprint(Set<URL> urls) {
-        List<String> listOfUrls = urls.stream().map(Object::toString).collect(Collectors.toList());
-        StringBuffer urlBuffer = new StringBuffer();
+    public static String generateAdditionalUrlsFingerprint(final Set<URL> urls) {
```

Comment:
Similar comment as above. This changes the behavior of a public method.

Reply:
As mentioned in the more lengthy comment, the return is still a string, although the content of the string is now based on SHA-256 instead of MD5.  Since the method name does not imply MD5, I'm not sure if it constitutes a change in semantic behavior.